### PR TITLE
[UI] rename MeshModel Summary to Registry in Settings

### LINKED ISSUE #9043: [UI] Settings: Rename "MeshModel Browser" to "Registry"
### Request

1. Change the menu label and the menu id/name/anchor to "Registry"
2. Find and update any references to this anchor/tab throughout the rest of Meshery UI.

Screenshot 2023-10-07 at 6 22 44 PM

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Adaminite**
**Notes for Reviewers**

This PR fixes #9043 

Just to make sure, please check that I didn't miss any anchor references. I don't think I found any other references outside of what I changed.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Adaminite** [comment]
I'm confused as to why the DCO failed. My commit message does have the sign off.

**Comment by Adaminite** [comment]
Commented out unused variables in custom-column.js. Opted to not delete because they may be used, but may need further review.

**Comment by leecalcote** [comment]
> Commented out unused variables in custom-column.js. Opted to not delete because they may be used in the future, but may need further review.

Thank you. Yes, they definitely need to be addressed.

File: ui/components/Dashboard/DashboardMeshModelGraph.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function MeshModelContructs({ classes }) {
 
   return (
     <div className={classes.dashboardSection}>
-      <Link href="/settings#meshmodel-summary">
+      <Link href="/settings#registry">
```
Comment (leecalcote) [inline ui/components/Dashboard/DashboardMeshModelGraph.js:75]: Good

File: ui/utils/custom-column.js [hunk 2]
```diff
@@ -29,10 +29,15 @@ const styles = (theme) => ({
 
 const CustomColumnVisibilityControl = ({ columns, customToolsProps, classes }) => {
   const [anchorEl, setAnchorEl] = useState(null);
+
+  /* 
+  ESLint error: values are never used
```
Comment (leecalcote) [inline ui/utils/custom-column.js:34]: FYI @sudhanshutech, please address.

**Review by leecalcote — APPROVED** [review]
LGTM.

Thank you, @Adaminite 👍

**Comment by Adaminite** [comment]
@leecalcote The UI E2E tests failed. Appears to be due to a timeout. Is it related to the change I made?

**Comment by leecalcote** [comment]
> @leecalcote The UI E2E tests failed. Appears to be due to a timeout. Is it related to the change I made?

Thanks for asking, @Adaminite. No, it's not related to the changes that you've made. Our e2e tests are flaky, generally causing more churn than they do help. We're trying to change that but have no one championing a move to Playwright. // @MarioArriaga92

**Comment by leecalcote** [comment]
> I'm confused as to why the DCO failed. My commit message does have the sign off.

They both do, yes. 🤔

**Comment by leecalcote** [comment]
@Adaminite have we found you your next issue?

**Comment by Adaminite** [comment]
@leecalcote Not yet. I can probably take a look at some of the beginner issues later and see what I can do. Are there any major tasks on either the UI or the main Meshery available?

**Pull request merged**